server/the-usher-openapi-spec.yaml

```diff
@@ -431,6 +431,47 @@ paths:
         404:
           $ref: '#/components/responses/NotFound'
 
+  /personas:
+    post:
+      'x-swagger-router-controller': 'personas/persona'
+      summary: Create a single persona
+      operationId: createPersona
+      tags:
+        - Admin APIs
+      security:
+        - bearerAdminAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                tenant_key:
```

Comment:
@k1ch  did we ever answer the open question on if we were going to add a `tenant_id` to the tenants table that would function similar to `client_id` in the clients table? 

If the answer is yes, then wouldn't this API accept `tenant_id` instead of `key` ?

Reply:
I just checked the DB schemas, we can use `tenant_name` and `client_id` since both have unique constraint.
I will update accordingly.

Reply:
@k1ch  I was thinking that we would *add* a `tenant_id` to the tenants table that would be the unique "url slug" for a tenant, and then rename the `tenant_name` field to be just `name` and it would be a human readable value, like what we are doing for clients. 

As an aside, I don't think that `client_id` needs to be involved in this API at all, I was using that as a reference.

Reply:
@k1ch after giving this a bit more thought, since my suggestion isn't possible yet since the new `tenant_id` field doesn't even exist yet, I am thinking we actually stick with your current implementation. 

Then, if we decide to add `tenant_id` in a separate PR, as part of that body of work we can refactor this API method. As long as we do that before we publish the `2.0.0` version this should be fine.

Reply:
In the request body we can have `tenant_name` instead of `tenant_key` since `name` has unique constraint.
```sql
CONSTRAINT tenants_name_uq UNIQUE (name)
```
Do you think we should use name until we introduce `tenant_id`?

Reply:
After a discussion with @k1ch  we decided to stick with the current implementation in this PR that uses the tenant key. We are putting on hold the concept of `tenant_id`.